entrypoint.sh

```diff
@@ -26,18 +26,16 @@ test_vars() {
 
 aws() {
 test_vars AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY
-set -x
 
-if [[ "${PODVM_LAUNCHTEMPLATE_NAME}" ]]; then
-	optionals+="-use-lt -aws-lt-name ${PODVM_LAUNCHTEMPLATE_NAME} "
-else
-	optionals+="-imageid ${PODVM_AMI_ID} "
-	optionals+="-instance-type ${PODVM_INSTANCE_TYPE:-t3.small} "
-	optionals+="-securitygroupid ${AWS_SG_ID} "
-	optionals+="-keyname ${SSH_KP_NAME} "
-	optionals+="-subnetid ${AWS_SUBNET_ID} "
-fi
+[[ "${PODVM_LAUNCHTEMPLATE_NAME}" ]] && optionals+="-use-lt -aws-lt-name ${PODVM_LAUNCHTEMPLATE_NAME} " # has precedence if set
+[[ "${AWS_SG_ID}" ]] && optionals+="-securitygroupid ${AWS_SG_ID} " # MUST if template is not used
```

Comment:
The short name be changed should also be changed to `securitygroupids`  include the variable AWS_SG_IDS